Content.Shared/Atmos/Piping/Binary/Components/GasVolumePumpComponent.cs

```diff
@@ -42,4 +42,10 @@ public sealed partial class GasVolumePumpComponent : Component
 
     [DataField]
     public float LastMolesTransferred;
+
+    /// <summary>
+    /// Frontier - Start the pump with the map.
+    /// </summary>
+    [DataField]
+    public bool StartOnMapInit;
```

Comment:
Consistency with `GasFilterComponent` and `GasMixerComponent.cs`

```suggestion
    public bool StartOnMapInit { get; set; } = false;
```